tokens/src/impls.rs

```diff
@@ -67,6 +67,10 @@ where
 			B::can_withdraw(asset, who, amount)
 		}
 	}
+
+	fn asset_exists(_asset: Self::AssetId) -> bool {
+		true
```

Comment:
```suggestion
		TestKey::contains(&asset)
```

Reply:
Every asset that is not in `TestKey` (`B`), is delegated to `A`, I think that this implementation will be not consistent with other methods?

Reply:
I think `unimplemented!()` is better

Reply:
I propose returning false for now, just to be on the safe side, it would be bad if it caused a runtime panic.